srcpkgs/poweralertd/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'poweralertd'
+pkgname=poweralertd
+version=0.2.0
+revision=1
+build_style=meson
+configure_args="-Dman-pages=enabled"
+hostmakedepends="pkg-config meson scdoc"
```

Comment:
Explicitly adding meson to hostmakedepeds is unnecessary, the meson build-style already does this.